chain/chain/src/types.rs

```diff
@@ -418,7 +409,23 @@ pub trait RuntimeAdapter: EpochManagerAdapter + Send + Sync {
         prev_epoch_last_block_hash: &CryptoHash,
         epoch_id: &EpochId,
         next_epoch_id: &EpochId,
-    ) -> Result<CryptoHash, Error>;
+    ) -> Result<CryptoHash, Error> {
```

Comment:
BTW - not sure why you called the PR "simplification"

Reply:
This is a red diff which removes the code. Implementing `get_epoch_sync_data_hash` once instead of overriding it twice is simpler. The logic is self is kept intact indeed.